docs/features/writing-different-telemetry-types.md

```diff
@@ -75,6 +76,45 @@ using (var measurement = DependencyMeasurement.Start())
 
 Note that we have an `LogServiceBusTopicDependency` to log dependency logs for an Azure Service Bus Topic and an `LogServiceBusDependency` to log Azure Service Bus logs where the entity type is not known.
 
+### Measuring Azure Table Storage Dependencies
+
+We allow you to measure Azure Table Storage dependencies.
+
+Here is how you can report a dependency call:
+
+```csharp
+var telemetryContext = new Dictionary<string, object>
+{
+    { "Tenant", "Contoso" },
+    { "Order", "ABC" },
+};
+
+var durationMeasurement = new Stopwatch();
+
+// Start measuring
+durationMeasurement.Start();
+var startTime = DateTimeOffset.UtcNow;
+
+_logger.LogTableStorageDependency(tableName: "orders", accountName: "orderAccount", isSuccessful: true, startTime, durationMeasurement.Elapsed, telemetryContext);
```

Comment:
I'm going to be a pain, but asking for account and then table makes more sense as it goes from least to most specific

Reply:
Also how it's typically done in SDKs/samples, etc.

Reply:
Thanks we have R#.